third_party/silabs/efr32_sdk.gni

```diff
@@ -200,6 +200,9 @@ template("efr32_sdk") {
     }
 
     if (defined(invoker.chip_enable_wifi) && invoker.chip_enable_wifi) {
+      if (enable_dic) {
+        defines += [ "DIC_ENABLE=1" ]
```

Comment:
This seems to duplicates the one present in `examples/platform/silabs/efr32/BUILD.gn`